packages/relay/src/lib/eth.ts

```diff
@@ -646,23 +647,24 @@ export class EthImpl implements Eth {
   async sendRawTransaction(transaction: string): Promise<string | JsonRpcError> {
     this.logger.trace('sendRawTransaction(transaction=%s)', transaction);
 
+    const parsedTx = ethers.utils.parseTransaction(transaction);
```

Comment:
Actually I would prefer an approach where `ethers` stays in the `precheck` file.
Instead could we update `precheck` with helper methods for each of the calling `eth_x` methods that encompasses the needed prechecks.

Example design below

```
  async sendRawTransactionCheck(transaction: string) {
    const parsedTx = Precheck.parseTxIfNeeded(transaction);
    this.gasLimit(parsedTx);
    await this.nonce(parsedTx);
    this.chainId(parsedTx);
    this.value(parsedTx);
  }
```